fuzz-tests/src/bin/decimal.rs

```diff
@@ -0,0 +1,40 @@
+#![cfg_attr(feature = "libfuzzer-sys", no_main)]
+
+use arbitrary::Arbitrary;
+#[cfg(feature = "libfuzzer-sys")]
+use libfuzzer_sys::fuzz_target;
+
+use radix_engine_common::math::*;
+
+#[derive(Debug, Arbitrary)]
+struct OneDecimal(Decimal, Decimal, i64, u32, i32, RoundingMode);
+
+// Fuzzer entry points
+#[cfg(feature = "libfuzzer-sys")]
+fuzz_target!(|decimal: OneDecimal| {
```

Comment:
Thinking if it would make sense to increase somehow a chance to hit corner cases, eg. Decimal::MAX, Decimal::MIN